_posts/2020-05-04-introducing-open-sgid.md

```diff
@@ -24,20 +24,20 @@ The timing could not be more perfect with the sudden increase in teleworking, wh
 
-A nearly month-long beta testing period was helpful to identify several issues and some inherent differences between the Open SGID and the enterprise geodatabase of the SGID10 era. A [GitHub repository](https://github.com/agrc/open-sgid) was created as the primary source of information for the Open SGID and to catalog and address [these issues and differences](https://github.com/agrc/open-sgid/issues) with explanations or troubleshooting techniques.
+A nearly month-long beta testing period was helpful to identify several issues and some inherent differences between the Open SGID and the enterprise geodatabase of the SGID10 era. A [GitHub repository](https://github.com/agrc/open-sgid) was created as the primary source of information for the Open SGID and to catalog and address [these issues and differences](https://github.com/agrc/open-sgid/issues) with explanations or troubleshooting techniques, or both. Tags are assigned to each issue to provide a level of categorization. 
 
 #### Known Issues
 
-- [Tables do not contain metadata](https://github.com/agrc/open-sgid/issues/5)
-- [Fields with domains currently only display coded values](https://github.com/agrc/open-sgid/issues/7)
+- [Tables do not contain metadata.](https://github.com/agrc/open-sgid/issues/5)
+- [Fields with domains currently only display coded values.](https://github.com/agrc/open-sgid/issues/7)
 
 #### Differences from SQL SDE Database
 
 - [Table names and casing](https://github.com/agrc/open-sgid/issues/12)
 - [Attribute names and casing](https://github.com/agrc/open-sgid/issues/13)
 - [Null value sorting behavior](https://github.com/agrc/open-sgid/issues/6)
-- [ArcMap requires selection of unique ID field](https://github.com/agrc/open-sgid/issues/9)
+- [Required  selection of unique ID field in ArcMap](https://github.com/agrc/open-sgid/issues/9)
 
-Tags are assigned to each issue to provide a level of categorization. This repository will be an important reference for Open SGID users to search for information about known issues, and to submit new issues they encounter, so AGRC staff and the general user population can collectively offer feedback and solutions. Use the green New issue button on the [issues page](https://github.com/agrc/open-sgid/issues) to submit yours, and visit this page when you first come across a problem with, or question about, the Open SGID to see if others have already found a solution.
+If you come across a problem with or have a question about the Open SGID, visit the [issues page](https://github.com/agrc/open-sgid/issues). This repository not only acts as an important resource for Open SGID users to search for information about known issues, but also allows users to submit new issues they encounter, so AGRC staff and the general user population can collectively offer feedback and solutions. If you have not yet found an answer or solution to your topic, you can use the green _New issue_ button to submit it to the community.
```

Comment:
```suggestion
If you come across a problem with or have a question about the Open SGID, visit the [issues page](https://github.com/agrc/open-sgid/issues). This repository not only acts as an important resource for Open SGID users to search for information about known issues, but also allows users to submit new issues they encounter, so AGRC staff and the general user population can collectively offer feedback and solutions. If you have not yet found an answer or solution to your topic, you can use the green _New issue_ button to submit it to the community. Also be sure to check out the [Open SGID Policy and Terms of Use page]({% /about/policy/open-sgid/ %}), which provides additional information. 

```

Reply:
Ps, not sure whether I did the link to the Terms page correctly, @steveoh . I tried!